StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/AccountSessionClaimedMessageHandler.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  AccountSessionClaimedMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/14/24.
+//
+
+import Foundation
+
+// Emitted when the account session has been created.
```

Comment:
supernit: Want to make these docstrings? (same comment applies to the other files)

Also, I don't know if "created" is interchangeable with "claimed", so it might be more accurate to use the same wording in the doc
```suggestion
/// Emitted when the claim response is available
```